code/script.js

```diff
@@ -1,47 +1,183 @@
           <p>${message}</p>
         </div>
       </section>
     `
+    }
+    // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
+    chat.scrollTop = chat.scrollHeight
+};
+
+
+// Sequency of QUESTION/ANSWER function:
+const nextQuestion = (message) => {
+  if (questionNumber === 1) {
+    userMessage(message)
+    // After the message is sent, input value becomes empty again.
+    input.value = ''
+    // Next bot question apears in 1s
+    setTimeout(() => allServices(message), 1000)
+  } else if (questionNumber === 2) {
+    userMessage(message)
+    setTimeout(() => serviceChoice(message), 1000)
+  } else if (questionNumber === 3) {
+    userMessage(message)
+    setTimeout(() => priceMessage(message), 1000)
+  } else if (questionNumber === 4) {
+    userMessage(message)
+    setTimeout(() => goodbyeMessage(), 1000)
   }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
-  chat.scrollTop = chat.scrollHeight
 }
 
-// Starts here
+// Starts here - BOT MESSAGES
+// First message
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  questionNumber = 1;
+  botMessage(`Happy to see you! <br> What's your name?`)
+}
+
+// Second message
+const allServices = (usersName) => {
+  questionNumber = 2;
+  botMessage(`Nice to meet you ${usersName}. What service are you interested in?`)
+
+  // Adding buttons for the user to choose the service
+  inputWrapper.innerHTML = `
+  <button id="hairBtn">Hair services</button>
+  <button id="nailBtn">Nail services</button>
+  `
+  // Adding event listeners to "catch" when the user clicks the button
+  document.getElementById('hairBtn').addEventListener('click', () => nextQuestion('Hair service'));
+  document.getElementById('nailBtn').addEventListener('click', () => nextQuestion('Nail service'));
 }
 
-// Set up your eventlisteners here
+// Third message 
+const serviceChoice = (choice) => {
+  questionNumber = 3;
+
+  botMessage(`Great choice! Now you can select in which ${choice} are you interested in`)
+
+  if (choice === 'Hair service') {
+
+    inputWrapper.innerHTML = `
+    <button id="hairCut">Hair Cut</button>
+    <button id="hairStyle">Hair Style</button>
+  `
+  document.getElementById('hairCut').addEventListener('click', () => nextQuestion('Haircut'));
+  document.getElementById('hairStyle').addEventListener('click', () => nextQuestion('Hairstyle'));
+  
+  } else if (choice === 'Nail service') {
+
+    inputWrapper.innerHTML = `
+    <button id="gelNailPolish">Gel nail polish</button>
+    <button id="manicure">Manicure</button>
+  `
+    document.getElementById('gelNailPolish').addEventListener('click', () => nextQuestion('Gel nail poslish'));
+    document.getElementById('manicure').addEventListener('click', () => nextQuestion('Manicure'));
+  }
+}
```

Comment:
All your messages functions look very close to each other. Maybe it is a good idea to create a reusable template function very similar to `showMessage`?